Docs/ProjectSpec.md

```diff
@@ -260,21 +260,33 @@ A source can be provided via a string (the path) or an object of the form:
 	- `sources` - Compile Sources phase
 	- `resources` - Copy Bundle Resources phase
 	- `headers` - Headers Phase
-	- `none` - Will not be added to any build phases
```

Comment:
Has none been removed?

Reply:
No. I’m not sure how that happened. I’ll fix it.